src/Infrastructure/DependencyInjection.cs

```diff
@@ -20,16 +21,23 @@ public static class DependencyInjection
         /// <returns></returns>
         public static IServiceCollection AddInfrastructure(this IServiceCollection services, IConfiguration configuration)
         {
-            services.AddDbContext<ApplicationDbContext>(options =>
-               options.UseNpgsql(
-                   configuration.GetConnectionString("DefaultConnection"),
-                   b => b.MigrationsAssembly(typeof(ApplicationDbContext).Assembly.FullName)));
+            services.AddDbContext<ApplicationDbContext>(async (provider, options) =>
```

Comment:
`async void`? This won’t work. `AddDbContext(…)` configures options, and options configuration is synchronous. If you have an async process here to retrieve the configuration from, then this won’t be able to synchronously configure the database context.

Reply:
Switched to `.GetAwaiter().GetResult()` to make it sync. Testing async with sync was not a good idea.